x-pack/platform/plugins/shared/index_management/public/plugin.ts

```diff
@@ -144,6 +147,20 @@ export class IndexMgmtUIPlugin
     return {
       apiService: new PublicApiService(coreSetup.http),
       extensionsService: this.extensionsService.setup(),
+      indexManagementApp: async (params: IndexManagementAppMountParams) => {
```

Comment:
lets call this `renderIndexManagementApp`

Reply:
updated in https://github.com/elastic/kibana/pull/209599/commits/b913415049a7c38edfae720a9e6d6325ff14498b